cmake_modules/VCMI_lib.cmake

```diff
@@ -184,6 +184,7 @@ macro(add_main_lib TARGET_NAME LIBRARY_TYPE)
 
 		${MAIN_LIB_DIR}/vstd/StringUtils.cpp
 
+                ${MAIN_LIB_DIR}/ArtifactUtils.cpp
```

Comment:
1) Use tabs, not spaces.
2) Add header in list below this one, to include it into IDE projects